templates/weave/1.9.3.tmpl.yaml

```diff
@@ -0,0 +1,243 @@
+apiVersion: v1
```

Comment:
Can you explain why this was added and I why I might want this?

Reply:
Similar to the ingress yaml, this is largely for pinning versions. It can be skipped by setting `SKIP_WEAVE=true`.